android-svc-lib.sh

```diff
@@ -378,11 +472,26 @@ SetShellType () {
     fi
 }
 
+###
+# Select a specific android device by serial id
+# ARGUMENTS:
+#  $1 serial
+# RETURN:
+#  nothing or exit the script with an error message 
 SetAdbDevice () {
     g_adbSerial="${1-}"
     [ -n "$g_adbSerial" ] || Exit 1 "Adb device was not provided in SetAdbDevice"
 }
 
+###
+# Execute an adb shell command.
+# GLOBALS:
+#  g_shellType defined by SetShellType only adb is currently supported
```

Comment:
`only adb is currently supported` - no if you leave it empty, it will default to using `su` instead